### PR TITLE
Add synthetic token price and pair tracking

### PR DESCRIPTION
**Pull request by NikolaCBS**

### CODE REVIEW

File: src/modules/icons/icons.service.ts
Comment (DanijelCBS) [inline src/modules/icons/icons.service.ts:1]: This cron should stay 3 hours and remove comment

File: src/modules/pairs/pairs-locker.sync.ts
Comment (DanijelCBS) [inline src/modules/pairs/pairs-locker.sync.ts:1]: This cron should stay 2 minutes and remove comment

File: src/modules/pairs/pairs.sync.ts [hunk 1]
```diff
@@ -50,7 +55,8 @@ export class PairsSync {
     );
   }
 
-  @Cron(CronExpression.EVERY_3_MINUTES)
```
Comment (DanijelCBS) [inline src/modules/pairs/pairs.sync.ts:53]: Undo cron change and comment

File: src/modules/rewards/rewards.sync.ts
Comment (DanijelCBS) [inline src/modules/rewards/rewards.sync.ts:1]: Same for cron and comment

File: src/modules/token-price/token-locker.sync.ts
Comment (DanijelCBS) [inline src/modules/token-price/token-locker.sync.ts:1]: Same

File: src/modules/token-price/token-price.sync.ts [hunk 2]
```diff
@@ -3,11 +3,15 @@ import { Cron, CronExpression } from '@nestjs/schedule';
...
+import { FPNumber } from '@sora-substrate/math';
+
 import { TokenPriceBcDto } from './dto/token-price-bc.dto';
 import { TokenPriceService } from './token-price.service';
+
+import { PROVIDER } from '../../constants/constants';
+
 import * as whitelist from 'src/utils/files/whitelist.json';
-import { FPNumber } from '@sora-substrate/math';
+import * as synthetic from 'src/utils/files/synthetics.json';
```
Comment (DanijelCBS) [inline src/modules/token-price/token-price.sync.ts:14]: Name it synthetics in plural

File: src/modules/token-price/token-price.sync.ts [hunk 3]
```diff
@@ -30,46 +34,66 @@ export class TokenPriceSync {
     );
   }
 
-  @Cron(CronExpression.EVERY_MINUTE)
+  // Every minute
```
Comment (DanijelCBS) [inline src/modules/token-price/token-price.sync.ts:37]: Same issue for cron

File: src/modules/tracker/tracker-pswap.sync.ts [hunk 4]
```diff
@@ -29,7 +29,8 @@ export class TrackerPswapSync {
     );
   }
 
-  @Cron(CronExpression.EVERY_10_MINUTES)
+  // Every 10 minutes
+  @Cron(CronExpression.EVERY_12_HOURS)
```
Comment (DanijelCBS) [inline src/modules/tracker/tracker-pswap.sync.ts:33]: Same for cron

File: src/modules/tracker/tracker-supply.sync.ts [hunk 5]
```diff
@@ -20,7 +20,8 @@ export class TrackerSupplySync {
     private readonly tokenPriceService: TokenPriceService,
   ) {}
 
-  @Cron(CronExpression.EVERY_HOUR)
+  // Every hour
+  @Cron(CronExpression.EVERY_12_HOURS)
```
Comment (DanijelCBS) [inline src/modules/tracker/tracker-supply.sync.ts:24]: Cron

File: src/modules/tracker/tracker-val.sync.ts [hunk 6]
```diff
@@ -30,7 +30,8 @@ export class TrackerValSync {
     );
   }
 
-  @Cron(CronExpression.EVERY_HOUR)
+  // Every hour
+  @Cron(CronExpression.EVERY_12_HOURS)
```
Comment (DanijelCBS) [inline src/modules/tracker/tracker-val.sync.ts:34]: Cron

**Review by DanijelCBS — CHANGES_REQUESTED** [review]

File: src/modules/tracker/tracker-val.sync.ts (hunk 6 above)
Comment (NikolaCBS) [reply src/modules/tracker/tracker-val.sync.ts]: @DanijelCBS  I have resetted the cron timers to the previous values.

**Review by DanijelCBS — APPROVED** [review]

**Pull request merged**